src/sections/target/MolecularInteractions/index.js

```diff
@@ -0,0 +1,9 @@
+export const definition = {
+  id: 'interactions',
+  name: 'Molecular Interactions',
+  shortName: 'MI',
+  hasData: data => true,
```

Comment:
From the schema, it looks like interactions could come back as null or have a count of 0. Probably would be a good idead to check for that here.